cues.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+---
+
+# Cues
+
+## Introduction
+
+The `Cues Element` provides an index of certain `Cluster Elements` to allow for optimized seeking to absolute timestamps within the `Segment`. The `Cues Element` contains one or many `CuePoint Elements` which each MUST reference an absolute timestamp (via the `CueTime Element`), a `Track` (via the `CueTrack Element`), and a `Segment Position` (via the `CueClusterPosition Element`). Additional non-mandated Elements are part of the `CuePoint Element` such as `CueDuration`, `CueRelativePosition`, `CueCodecState` and others which can improve seeking performance in some Matroska players.
```

Comment:
You're talking about "some players" here. To me it sounds a bit backwards: "the elements exist because there are players that can use them". I'd rather we state that the elements exist to give any potential reader more information to work with. Maybe something like "…and others which provide more information for seeking decisions in certain situations"?